acme/acme/client.py

```diff
@@ -150,7 +150,7 @@ def deactivate(self, regr):
         new_regr = self._regr_from_response(
             response, uri=regr.uri, new_authzr_uri=regr.new_authzr_uri,
             terms_of_service=regr.terms_of_service)
-        if response.status_code != 202:
+        if response.status_code not in (200, 202):
```

Comment:
Boulder currently returns 200 but it is supposed to return 202. See [boulder #2313](https://github.com/letsencrypt/boulder/issues/2313).